docs/WritingTests.asciidoc

```diff
@@ -1192,3 +1195,48 @@ the clone source job or specifying the variable explicitly:
 ----
 clone_job.pl --from https://openqa.opensuse.org  --host localhost 24 TESTDEBUG=1 SKIPTO=consoletest-yast2_i
 ----
+
+=== Defining a custom test schedule or custom test modules
+
+Normally the test schedule, that is which test modules should be executed and
+which order is prescribed by the +main.pm+ file within the test distribution.
+Additionally it is possible to exclude certain test modules from execution
+using the os-autoinst test variables +INCLUDE_MODULES+ and +EXCLUDE_MODULES+ as
+well as define a custom schedule using the test variable +SCHEDULE+.
+Also test modules can be defined and overridden on-the-fly using a
+downloadable asset.
+
+For example one can schedule a job on a production instance with a custom
+schedule consisting of two modules from the provided test distribution plus
+one test module which is defined dynamically and downloaded as an asset from
+an external trusted download domain:
+
+----
+openqa-clone-job --from https://openqa.opensuse.org --host https://openqa.opensuse.org 24 SCHEDULE=tests/boot/boot_to_desktop,tests/console/consoletest_setup,foo ASSET_1_URL=https://download.opensuse.org/my/test/foo.pm
```

Comment:
however users get things on download.opensuse.org/my

Reply:
yes, it's just an example. I could give http://example.org if you prefer

Reply:
it would be more obvious IMO

Reply:
done